src/docs/product/issues/issue-details/performance-issues/uncompressed-asset/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Uncompressed Asset"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/uncompressed-asset/
+description: "Learn more about Uncompressed Asset issues and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Uncompressed asset issues are when one of the files your users are downloading to load your browser page was not compressed when transferred. This likely indicates a misconfiguration of the server or CDN (content delivery network) serving the asset file.
+
+## Detection Criteria
+
+The detector for this performance issue looks for specific asset (resource) spans with the following criteria:
+
+- The span has to be one of either `resource.link` or `resource.script` spans, which in practice means it is a `.css` or `.js` file.
```

Comment:
From what I've seen it's almost always, but not necessarily always. I added this in because people don't really understand 'span ops' but if you say `css / js` I think it's clearer what it's detecting.